sitemap_cron.py

```diff
@@ -1,13 +1,22 @@
 # local
+import sys
 import sitemap
 import config
 from raven import Client
 from github_connection import GH
+from xml.sax.handler import ContentHandler
+from xml.sax import make_parser
 
 
 client = Client(config.SENTRY_URL)
 
 
+def parse_file(filename):
+    parser = make_parser()
+    parser.setContentHandler(ContentHandler())
+    parser.parse(filename)
```

Comment:
Does this method need to have a return value?

Reply:
The method is just used to check if the syntax is correct, so it doesn't need to return anything. If the method fails, then an exception is caught and sent out via sentry and the program stops.

Reply:
👍

Reply:
Might be worth a comment mentioning that

Reply:
Added a comment!